src/game.h

```diff
@@ -114,6 +114,7 @@ class game
         friend class editmap;
         friend class advanced_inventory;
         friend class DynamicDataLoader; // To allow unloading dynamicly loaded stuff
+        friend class player_activity; // To allow drop of items in player_activity::finish
```

Comment:
Nope, big nope.  Add a method to game that allows what you want, but there's no reason to weaken encapsulation like this.  For that matter, editmap and the data loader are rather 'special', but there's no good reason for advanced inventory to have special access to game either.